### DISCUSSION TITLE
Stop running program

### DISCUSSION THREAD

**Discussion by andrewg78** [BIPES/BIPES#95] [Q&A] [answered]
Stop running program - what is the action behind this button taken?
Should the program be immediately terminated when I click on it somewhere in the middle of the execution?

**Comment by rafaelaroca** [comment, marked as answer]
Hi @andrewg78 ! It simply sends a Ctrl+C to the console/board. It will stop the running program, in most cases, so yes, the program will be terminated immediately in the middle of execution.

**Comment by andrewg78** [comment]
Hi. I was asking because it doesn't work for me. Could you please check this on your end?
